### PR TITLE
Updated deprecated links in outgoing-webhooks _index.md

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

Updated deprecated links in outgoing-webhooks _index.md

#### Ticket Link

NA

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes if any and also add the appropriate labels for Docathon.

File: site/content/integrate/outgoing-webhooks/_index.md [hunk 1]
```diff
@@ -89,10 +89,10 @@ Outgoing webhooks support more than just the `text` field. Here is a full list o
 
 | Parameter | Description | Required |
 |---|---|---|
-| text | [Markdown-formatted](https://docs.mattermost.com/help/messaging/formatting-text.html) message to display in the post.<br> To trigger notifications, use `@<username>`, `@channel` and `@here` like you would in normal Mattermost messaging. | If `attachments` is not set, yes |
+| text | [Markdown-formatted](https://docs.mattermost.com/messaging/formatting-text.html) message to display in the post.<br> To trigger notifications, use `@<username>`, `@channel` and `@here` like you would in normal Mattermost messaging. | If `attachments` is not set, yes |
```
Comment (justinegeffen) [inline site/content/integrate/outgoing-webhooks/_index.md:92]:
```suggestion
| text | [Markdown-formatted](https://docs.mattermost.com/messaging/formatting-text.html) message to display in the post.<br> To trigger notifications, use `@<username>`, `@channel`, and `@here` like you would in normal Mattermost messaging. | If `attachments` is not set, yes |
```

File: site/content/integrate/outgoing-webhooks/_index.md [hunk 2]
```diff
@@ -125,6 +125,6 @@ See the [admin guide's notes on Slack compatibility](https://docs.mattermost.com
 
 ### Tips and Best Practices
 
-1. If the `text` is longer than the allowable character limit per post, the message is split into multiple consecutive posts, each within the character limit. Servers running Mattermost Server v5.0 or later [can support posts up to 16383 characters](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+1. If the `text` is longer than the allowable character limit per post, the message is split into multiple consecutive posts, each within the character limit. Servers running Mattermost Server v5.0 or later [can support posts up to 16383 characters](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
 2. Your webhook integration may be written in any programming language as long as it supports handling HTTP POST requests.
 3. Outgoing webhooks are supported in public channels only. If you need a trigger that works in a private channel or a direct message, consider using a [slash command](/integrate/slash-commands/) instead.
```
Comment (justinegeffen) [inline site/content/integrate/outgoing-webhooks/_index.md:130]:
```suggestion
3. Outgoing webhooks are supported in public channels only. If you need a trigger that works in a Private channel or a Direct Message, consider using a [slash command](/integrate/slash-commands/) instead.
```

**Review by justinegeffen — COMMENTED** [review]
Two minor editorial corrections.

**Comment by Rutam21** [comment]
@justinegeffen Changes are applied. Please review.

**Review by justinegeffen — APPROVED** [review]
Looks good to me, thank you!

**Review by justinegeffen — APPROVED** [review]
Thank you for your contribution! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**